lib/Doctrine/Migrations/Configuration/XmlConfiguration.php

```diff
@@ -41,7 +42,11 @@ protected function doLoad(string $file) : void
             throw XmlNotValid::failedValidation();
         }
 
-        $xml = simplexml_load_file($file, SimpleXMLElement::class, LIBXML_NOCDATA);
+        $xmlString = file_get_contents($file);
+        $xml       = false;
+        if ($xmlString !== false) {
```

Comment:
This should be an assertion instead: `assert($xmlString !== false)`. Also please name the variable i.e. `$rawXml`.

Reply:
Sure, changed that. Thanks for your time!